js/sider/DialogmoterSide.js

```diff
@@ -0,0 +1,227 @@
+    }
+    if (skalHenteToggles) {
+        actions.hentToggles();
+    }
+    finnOgHentManglendeOppfolgingsforlopsPerioder(oppfolgingsforlopsPerioderReducerListe, virksomhetsnrListe, actions.hentOppfolgingsforlopsPerioder);
+};
+
+class Container extends Component {
+    componentDidMount() {
+        const {
+            togglesReducer,
+            harForsoektHentetAlt,
+            skalViseMotebehov,
+        } = this.props;
+
+        hentRessurser(this.props);
+
+        if (togglesReducer.hentet && harForsoektHentetAlt && skalViseMotebehov === false) {
+            history.push(`${getContextRoot()}/dialogmoter/mote`);
+        }
+    }
+
+    componentWillReceiveProps(nextProps) {
+        const { togglesReducer } = this.props;
+        const { harForsoektHentetAlt } = nextProps;
+
+        hentRessurser(nextProps);
+
+        if (togglesReducer.henter && !nextProps.togglesReducer.henter && harForsoektHentetAlt && nextProps.skalViseMotebehov === false) {
+            history.push(`${getContextRoot()}/dialogmoter/mote`);
+        }
+    }
+
+    render() {
+        const {
+            brodsmuler,
+            henter,
+            hentingFeilet,
+        } = this.props;
+        return (<Side
+            tittel={getLedetekst('mote.moter.sidetittel')}
+            brodsmuler={brodsmuler}
+            laster={henter}>
+            {
+                (() => {
+                    if (henter) {
+                        return <AppSpinner />;
+                    } else if (hentingFeilet) {
+                        return <Feilmelding />;
+                    }
+                    return (<DialogmoterInnhold
+                        {...this.props}
+                    />);
+                })()
+            }
+        </Side>);
+    }
+}
+
+Container.propTypes = {
+    henter: PropTypes.bool,
+    hentingFeilet: PropTypes.bool,
+    ledetekster: keyValue,
+    brodsmuler: PropTypes.arrayOf(brodsmulePt),
+    koblingId: PropTypes.string,
+    motebehovReducer: motebehovProptypes.motebehovReducerATPt,
+    togglesReducer: togglesPt,
+    harMote: PropTypes.bool,
+    harForsoektHentetAlt: PropTypes.bool,
+    skalViseMotebehov: PropTypes.bool,
+    virksomhetsnrListe: PropTypes.arrayOf(PropTypes.string),
+    virksomhetnrMedMotebehovListe: PropTypes.arrayOf(PropTypes.string),
+    actions: PropTypes.shape({
+        hentDineSykmeldinger: PropTypes.func,
+        hentLedere: PropTypes.func,
+        hentMote: PropTypes.func,
+        hentMotebehov: PropTypes.func,
+        hentOppfolgingsforlopsPerioder: PropTypes.func,
+        hentToggles: PropTypes.func,
+    }),
+};
+
+export function mapDispatchToProps(dispatch) {
+    const actions = bindActionCreators({
+        hentDineSykmeldinger,
+        hentLedere,
+        hentMote: moteActions.hentMote,
+        hentMotebehov,
+        hentOppfolgingsforlopsPerioder,
+        hentToggles,
+    }, dispatch);
+
+    return {
+        actions,
+    };
+}
+
+export function mapStateToProps(state) {
```

Comment:
Dnene containeren/mapStateToProps-funksjonen gjør alt for mange ting... :-( Ville ha splittet den opp i flere biter.